provider/resources.go

```diff
@@ -6043,6 +6043,26 @@ func setupComputedIDs(prov *tfbridge.ProviderInfo) {
 	) (resource.ID, error) {
 		return attr(state, "arn"), nil
 	}
+	prov.Resources["aws_quicksight_role_membership"].ComputeID = func(
+		ctx context.Context, state resource.PropertyMap,
+	) (resource.ID, error) {
+		return attrWithSeparator(state, "role", "memberName"), nil
```

Comment:
```suggestion
		return attrWithSeparator(state, ",", "awsAccountId", "namespace", "role", "memberName"), nil
```

I think this should match what [upstream](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/quicksight_role_membership#import) uses for the id